src/solvers/base_solver.cpp

```diff
@@ -11,25 +11,24 @@
 
 BaseSolver::BaseSolver()
   : m_ess_bdr_coef(nullptr), m_nat_bdr_coef(nullptr), m_output_type(OutputType::VisIt),
-    m_timestepper(TimestepMethod::ForwardEuler), m_ode_solver(nullptr), m_time(0.0), m_cycle(0), m_visit_dc(nullptr),
-    m_gf_initialized(false)
+    m_time(0.0), m_cycle(0), m_visit_dc(nullptr), m_gf_initialized(false)
 {
-  m_ode_solver = new mfem::ForwardEulerSolver;
+  SetTimestepper(TimestepMethod::ForwardEuler);
 }
 
-BaseSolver::BaseSolver(mfem::Array<FiniteElementState> &state)
-  : m_state(state), m_ess_bdr_coef(nullptr), m_nat_bdr_coef(nullptr),
-    m_output_type(OutputType::VisIt),
-    m_timestepper(TimestepMethod::ForwardEuler), m_ode_solver(nullptr), m_time(0.0), m_cycle(0), m_visit_dc(nullptr),
-    m_gf_initialized(false)
-{
-  MFEM_ASSERT(state.Size() > 0, "State vector array of size 0 in BaseSolver constructor.");
-
-  MPI_Comm_rank(m_state[0].space->GetComm(), &m_rank);
-
-  m_ode_solver = new mfem::ForwardEulerSolver;
-
-}
+//BaseSolver::BaseSolver(std::map < std::string, mfem::Array<FiniteElementState> &state)
```

Comment:
Don't leave commented code.  It is either needed (and uncommented) or should be deleted.

Reply:
Good point. I stubbed it out just to get things compiling. It wasn't used in any of the tests and is kind of a cumbersome constructor anyway, so I'm leaning toward just removing it.